gradle.properties

```diff
@@ -15,6 +15,3 @@ org.gradle.jvmargs=-Xmx2048m
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
-
-# keep build timestamps in APK
-android.keepTimestampsInApk = true
```

Comment:
Hmm why do we need to remove this?

Reply:
The latest android studio version (4.0, I think?) refused to build with it in place, saying that it had previously been deprecated and was now removed and refused to build with it present.

Reply:
Ah alright